framework/src/problems/FEProblemBase.C

```diff
@@ -4721,6 +4721,10 @@ FEProblemBase::updateMaxQps()
   }
 
   unsigned int max_qpts = getMaxQps();
+  if (max_qpts > Moose::constMaxQpsPerElem)
```

Comment:
Why not move this into `bumpVolumeQRuleOrder`?

Reply:
Because we don't know what the new max qps is until here.  in bumpVolumeQRuleOrder - we only know what the new order is.